Dockerfile

```diff
@@ -5,7 +5,8 @@ RUN apk update && apk add --no-cache py3-pip \
     cmake=3.24.4-r0 \
     linux-headers=5.19.5-r0 \
     boost-build=1.79.0-r0 \
-    libtbb=2021.7.0-r0
+    libtbb=2021.7.0-r0 \
+    zstd-dev=1.5.5-r0
```

Comment:
Is there a reason why we don't install that via Conan? It seems to be available: https://conan.io/center/zstd

Reply:
At first, I added it only to the `conanfile.py`, but CMake it wouldn't find the package, this is still somewhat experimenting

Reply:
Is there a reason why we do not do the same for e.g. https://conan.io/center/tbb ?

Reply:
Yes. TBB complains a lot when you try to link it statically, since it depends too much on the individual CPU. So we decided to give in on that end.